src/precice/config/ParticipantConfiguration.cpp

```diff
@@ -164,8 +164,8 @@ ParticipantConfiguration::ParticipantConfiguration(
                                "processors. Both result in the same distribution (if the safety factor is sufficiently large). "
                                "\"on-master\" is not supported if you use two-level initialization. "
                                "For very asymmetric cases, the filter can also be switched off completely (\"no-filter\").")
-                           .setOptions({VALUE_FILTER_ON_MASTER, VALUE_FILTER_ON_SLAVES, VALUE_NO_FILTER})
-                           .setDefaultValue(VALUE_FILTER_ON_SLAVES);
+                           .setOptions({VALUE_FILTER_ON_PRIMARY, VALUE_FILTER_ON_SECONDARIES, VALUE_NO_FILTER})
```

Comment:
These filters should be backwards compatible too. As an example both `on-master` and `on-primary` should work.